json-endpoints/jetpack/class.jetpack-json-api-themes-modify-endpoint.php

```diff
@@ -64,4 +64,48 @@ function update() {
 		return true;
 	}
 
+	function update_translations() {
```

Comment:
Correct, this is from when I updated the PR from master, but then Sam rebased it to the older branch.  Awkward cruft.  I'm thinking it'll probably get smushed with everything else trimmed off before it's merged in.